src/modules/launcher/Plugins/Community.PowerToys.Run.Plugin.VSCodeWorkspaces/Properties/Resources.resx

```diff
@@ -118,14 +118,14 @@
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
   <data name="PluginTitle" xml:space="preserve">
-    <value>VSCode Workspaces</value>
+    <value>Visual Studio Code Workspaces</value>
   </data>
   <data name="In" xml:space="preserve">
     <value>in</value>
     <comment>Used to indicate the location where something is</comment>
   </data>
   <data name="PluginDescription" xml:space="preserve">
-    <value>Open VSCode previously opened workspaces, remote machines (SSH or Codespaces) and containers.</value>
+    <value>Opens VS Code previously opened workspaces, remote machines (SSH or Codespaces) and containers.</value>
```

Comment:
`Opens in VS Code previously` or `Opens VS Code's previously`? This sounds better for me. But im not native speaking